scout/adapter/mongo/variant_loader.py

```diff
@@ -658,85 +668,82 @@ def load_variants(
 
             LOG.debug("Attempt to load %s %s VCF.", variant_type, category.upper())
             variant_file = case_obj["vcf_files"].get(vcf_file_key)
+            if variant_file:
+                variant_files.append(variant_file)
 
-        if not variant_file:
+        if not variant_files:
             raise SyntaxError(
-                "VCF file {} {} does not seem to exist".format(category, variant_type)
+                "VCF files for {} {} does not seem to exist".format(category, variant_type)
             )
 
-        # Check if there are any variants in file
-        try:
+        for variant_file in variant_files:
+            if not self._has_variants_in_file(variant_file):
+                continue
+
             vcf_obj = VCF(variant_file)
-            var = next(vcf_obj)
-        except StopIteration as err:
-            LOG.warning("Variant file %s does not include any variants", variant_file)
-            return nr_inserted
-        # We need to reload the file
-        vcf_obj = VCF(variant_file)
```

Comment:
This is the same, only it's in a loop for each of the variant files found for a category, type pair.